product/model.py

```diff
@@ -281,9 +281,9 @@ def add_generic_dependency(self, generic_dependency):
 ## upload result model
 
 class UploadStatus(Enum):
-    SKIPPED_ALREADY_EXISTED = 1
-    UPLOADED_PENDING = 2
-    UPLOADED_DONE = 4
+    SKIPPED = 1
```

Comment:
yes, why not. how about pushing this commit to the master branch already, so the PR may focus around the actual "payload"?